packages/vite-plugin-cloudflare/playground/__test-utils__/index.ts

```diff
@@ -1,13 +1,18 @@
 import fs from "node:fs";
-import { version } from "vite";
+import semverGte from "semver/functions/gte";
+import { version as viteVersion } from "vite";
 import { onTestFinished, test } from "vitest";
 import { isWindows } from "../vitest-setup";
 
 export * from "../vitest-setup";
 export * from "./responses";
 
+export function satisfiesViteVersion(minVersion: string): boolean {
+	return semverGte(viteVersion, minVersion);
+}
+
```

Comment:
What is the reason for duplicating this rather than using the one in utils?
Is it that during tests we might have swapped out the vite version, and the one in utils gets hard coded into the bundled code?

Reply:
The one in utils won't be hard coded. We can use the same, I just generally prefer to avoid importing directly from parent packages as it makes the boundaries less clear. In fact I meant to add `semver` as a dev dependency of the playground package for the same reason. What do you think?

Reply:
https://github.com/cloudflare/workers-sdk/pull/11723/commits/9591c846ea68463c0230e84335b86ca87518fb81